src/components/procedure/_export-findings-csv.md

```diff
@@ -0,0 +1,36 @@
+You can export findings to a **CSV** file. Semgrep can export up to **10,000 most recent findings**. For findings greater than 10,000, you must use the [<i class="fas fa-external-link fa-xs"></i>API](https://semgrep.dev/api/v1/docs/).
+
+Findings exported in the CSV file are not affected by filters you apply in the page.
+
+Export findings by navigating to the product page and clicking the **<i class="fa-regular fa-download"></i> icon** near the time filters.
+
+![The download findings CSV button](/img/download-csv.png#md-width)
+_**Figure**. The download findings CSV button._
+
+<details>
+<summary>Click to view a description of all fields included in the CSV.</summary>
+| Field  | Description |
+| -------  | ------ |
+| Id | The unique id number of the finding. |
```

Comment:
Thanks for putting this in a table! We do have slightly different column values for each product

Reply:
Ok, I'll check the fields for all three!